packages/core/src/createActor.ts

```diff
@@ -178,6 +179,21 @@ export class Actor<TLogic extends AnyActorLogic>
           );
         }
         (child as any)._stop();
+      },
+      spawnChild: <T extends AnyActorLogic>(
+        logic: T,
+        actorOptions?: ActorOptions<T>
```

Comment:
this should reuse the type tricks from here: https://github.com/statelyai/xstate/blob/8a7ea9eb5faca407890cfa597eca2915660ab6bd/packages/core/src/createActor.ts#L763-L770